.clang-format

```diff
@@ -1,12 +1,5 @@
 ---
 BasedOnStyle: LLVM
-TabWidth: 4
-UseTab: Never
----
-Language:  JavaScript
-DisableFormat: true
----
-Language:        Cpp
```

Comment:
This introduces an obj-c++ file, which we want to use the same formatting for as c++. We don't have JS files in the repo any more so that part is outdated.